src/main/kotlin/com/msg/gcms/domain/attendance/presentation/AttendanceController.kt

```diff
@@ -71,4 +73,9 @@ class AttendanceController(
         response.setHeader("Content-Disposition", "attachment; filename=${URLEncoder.encode("$currentDate 출석부", "UTF-8").replace("+", "%20")}.xlsx")
         return clubAttendanceStatusExcelService.execute(currentDate)
     }
+
+    @GetMapping("/{club_id}/my")
+    fun findAttendSelfCheck(@PathVariable("club_id") id: Long): ResponseEntity<AttendSelfCheckResponseDto> =
```

Comment:
attendance 도메인이니까 id에 club id 라고 명시하는게 좋을 것 같아요
```suggestion
    fun findAttendSelfCheck(@PathVariable("club_id") clubId: Long): ResponseEntity<AttendSelfCheckResponseDto> =
```